File: leetcode problems/tempCodeRunnerFile.py
from typing import List


class Solution:
    def twoSum(self, numbers: List[int], target: int) -> List[int]:
        
        l = 0
        r = len(numbers)-1
        
        while True:
            if numbers[l] + numbers[r] == target:
                return [numbers[l], numbers[r]]
            elif numbers[l] + numbers[r] < target:
                l+=1
            else:
                r-=1

g = Solution()

print(g.twoSum([2, 7, 11, 15], 9))

File: cSharpLeetcodeProblems/searchInRotatedSortedArray/tempCodeRunnerFile.cs
public class Solution {

    static void Main(string[] args){
        Console.WriteLine("P");
    }
    public int Search(int[] nums, int target) {
        

        int l = 0;
        int r = nums.Length -1;

        while(l < r){
            int midpoint = l + (l-r)/2;

            if(nums[midpoint] > nums[r]){
                l = midpoint+1;
            }
            else{
                r = midpoint;
            }
        }
        Console.WriteLine("PivotIdx: " + l);


    }
}